docs/working-with-cdk-for-terraform/remote-state.md

```diff
@@ -0,0 +1,28 @@
+# Terraform Remote State
+
+[Terraform remote state](https://www.terraform.io/docs/providers/terraform/d/remote_state.html) retrieves state data from a [Terraform backend](https://www.terraform.io/docs/backends/index.html). This allows you to use the root-level outputs of one or more Terraform configurations as input data for another configuration.
+
+Typesciprt example usage:
+```typescript
+.....
+import { TerraformCloudRemoteState } from 'cdktf';
```

Comment:
I would rename this to `TerraformCloudRemoteBackend` as that is essentially what you are configuring when you specify the following:

```
{
   organization: 'hashicorp',
   workspaces: {
       name: 'vpc-prod'
    }
}
```

Reply:
@anubhavmishra and I just talked about it, there was a bit confusion around the naming here. Will elaborate in a comment on the PR itself.